_data/developer.yml

```
@@ -403,6 +403,12 @@ websites:
       - totp
     doc: https://github.com/composer/packagist/pull/1031
 
+  - name: Pagerduty
+    url: https://pagerduty.com
+    img: pagerduty.png
+    twitter: pagerduty
```

@PagerDutyHelp is actually better suited for this 😅 
```suggestion
    twitter: PagerDutyHelp
```

---

Resolved from pagerduty to PagerDutyHelp ...Can I get an approval on this PR @Carlgo11?